kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -797,6 +797,14 @@ object KyuubiConf {
     .checkValues(AuthTypes)
     .createWithDefault(Seq(AuthTypes.NONE.toString))
 
+  val RESTFUL_AUTHENTICATION: ConfigEntry[Boolean] =
+    buildConf("kyuubi.restful.security.enabled")
```

Comment:
such a flag is too crude

Reply:
agree. Now I just turn off RESTful API authentication. Is there anything else I need to consider or implement?

Reply:
THRIFT-HTTP and RESTful are different things, your change affects both.

```
kyuubi.authentication=
// fallback to `kyuubi.authentication` if not configure
kyuubi.frontend.mysql.authentication=
kyuubi.frontend.rest.authentication=
```

Reply:
agree 👍. Isolated authentication method configuration is better.